content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -172,7 +172,7 @@ For a reference to old feature gates that are removed, please refer to
 | `PodDeletionCost` | `true` | Beta | 1.22 | |
 | `PodDisruptionConditions` | `false` | Alpha | 1.25 | 1.25 |
 | `PodDisruptionConditions` | `true` | Beta | 1.26 | |
-| `PodHasNetworkCondition` | `false` | Alpha | 1.25 | |
```

Comment:
Please **move** this line to `feature-gates-removed.md`.
There could be users who were/are actively using it.

Reply:
Done.